src/routes/publicRoutes.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+
+const publicRoutes = [<Route exact path="/" />, <Route exact path="/megaStrona" />, <Route exact path="/signIn" />];
```

Comment:
const publicRoutes = [<Route exact path="/" />, <Route exact path="/megaStrona" />, <Route exact path="/signIn" />];

it should be wrapped